doc/sphinx/addendum/type-classes.rst

```diff
@@ -274,23 +274,41 @@ This declares singleton classes for reflexive and transitive relations,
 (see the :ref:`singleton class <singleton-class>` variant for an
 explanation). These may be used as parts of other classes:
 
+.. coqtop:: none
+
+   Set Warnings "-future-coercion-class-field".
+
 .. coqtop:: all
 
    Class PreOrder (A : Type) (R : relation A) :=
      { PreOrder_Reflexive :> Reflexive A R ;
        PreOrder_Transitive :> Transitive A R }.
 
+.. coqtop:: none
+
+   Set Warnings "+future-coercion-class-field".
```

Comment:
FYI: you can also use `.. coqtop:: reset` here.  Of course, you lose any definitions/settings/theorems above that point.